app/client/ui/DescriptionConfig.ts

```diff
@@ -23,22 +25,22 @@ export function buildDescriptionConfig(
     // update a different column.
     let editor: HTMLTextAreaElement | undefined;
     owner.autoDispose(
-      cursor.subscribe(() => {
+      options.cursor?.subscribe(() => {
```

Comment:
```suggestion
      options.cursor.subscribe(() => {
```

Reply:
If we put the cursor in the `options` object it's because it's optional no ? Otherwise I don't understand why we would want to do this

Reply:
It's not optional currently; if we don't pass in a cursor, we'll get a type error. It's ok for us to have required properties in `options` though.

The reason why I suggested the change is because we had too many parameters (more than 3).